api/src/couchdb/index.ts

```diff
@@ -145,53 +152,116 @@ export const getInvitesDB = (): PouchDB.Database | undefined => {
   return _invitesDB;
 };
 
-export const getProjectMetaDB = async (
+/**
+ * Gets a database client object for the metadata DB for a given project document
+ * @param projectDoc The project document to use as a connection reference for the metadata DB
+ * @returns The connected instantiated database
+ */
+export const getMetadataDbFromProjectDocument = async (
```

Comment:
You've introduced this new export to use it only in the delete notebook method which could get the dbs from the project id instead.  I think it would be simpler just to access the dbs through the project id as before. Introducing another way just means you need longer funciton names to disambiguate them. I don't see any improvement by adding this.

Reply:
The benefit was just for performance - reduces the round-trips to DB. There were flows in the code which were

- get project doc
- validate something about it 
- get metadata db (which gets project doc again)

With this change you can just do 

- get project doc
- validate something about it 
- get metadata db from project doc

Minor improvement though

Reply:
Reverted to previous version, retaining additional documentation.